webapp-django/crashstats/crashstats/static/crashstats/js/socorro/nav.js

```diff
@@ -1,48 +1,32 @@
 $(document).ready(function () {
-    var product,
-        product_version,
-        report;
+    var reportsURLs = {};
+    $('#report_select option').each(function(i, elem) {
+        reportsURLs[elem.value] = {
+            'product': elem.dataset.urlProduct,
+            'version': elem.dataset.urlVersion,
+        };
+    });
```

Comment:
Why make it a function? Because it's only relevant if someone triggers a change event on `.version-nav select` below.